apps/site/assets/ts/projects/components/__projects.d.ts

```diff
@@ -15,7 +15,44 @@ export interface Image {
 }
 
 export interface Route {
-  mode: string;
+  mode: string | null;
   id: string;
   group: string;
 }
+
+export interface Link {
+  alt: string;
+  url: string;
+}
+
+export interface Banner {
+  blurb: string;
+  link: Link;
+  thumb: Image;
+  banner_type: "default" | "important";
+  text_position: "left" | "right";
+  category: string;
+  routes: Route[];
+  title: string;
+  updated_on: string;
+  utm_url: string;
+}
+
+export interface ProjectTeaser {
```

Comment:
This `interface` seems redundant with `Project` above.

Reply:
I can fix this up.

ProjectTeaser would be the "pure" CMS API interface for a teaser of type project whereas Project above shows the version we're sending down which is "simplified". I like having the Elixir structs match the Typescript interfaces but this already is polluted by the simplified version via optional parameters. We do have like Simple<Blah> interfaces so maybe I'll rename Project to SimpleProject.